test-plone-4.3.x.cfg

```diff
@@ -7,3 +7,9 @@ package-name = ftw.contacts
 
 [versions]
 Products.LDAPUserFolder = <=3a
+
+# collective.geo.openlayers >= 4.0 is for Plone 5.
+collective.geo.openlayers = 3.2b1
```

Comment:
Could also make something like:
```
collective.geo.openlayers = <=4a
```
if you want to be less specific 😉

Reply:
Thanks for the hint...